src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -395,10 +395,8 @@ void manualRegistrationToolBox::reset()
     setDisableComputeResetButtons(true);
 }
 
-void manualRegistrationToolBox::hideEvent(QHideEvent *event)
+void manualRegistrationToolBox::clear()
 {
-    medAbstractSelectableToolBox::hideEvent(event);
```

Comment:
In fact, `hideEvent` is automatically done. When we declare it here, it's just to add some functionnalities :)